"""
try:
    # Operaci贸n riesgosa
except TipoDeExcepci贸n:
    # Manejo de la excepci贸n
else:
    # C贸digo que se ejecuta si y solo si no hay excepciones

"""


def dividir_dos_numeros(numero1: int, numero2: int) -> int:
    return numero1 / numero2


try:
    print(dividir_dos_numeros(5, 2))
except ZeroDivisionError:
    print("no dividir por cero")
else:
    print("Estoy aqui!")
